_kbarticles/2022-05-05-creating-postgresql-databases-from-shared-rds.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Creating PostgreSQL databases from a shared RDS instance using postgres-tinsmith"
+date: May 5, 2022
+excerpt: How to use postgres-tinsmith service broker to quickly create PostgreSQL databases from a shared RDS instance
+---
+
+The cloud.gov team [recently deprecated the shared database instance plans]({{ site.baseurl }}{% link _posts/2022-04-25-deprecation-notice.md %}) of our platform.
+
+One of the major benefits of the shared database instance plans was the ability to very quickly create new
+databases, facilitating rapid application development. Unlike dedicated instance plans, which each provision a
+separate RDS database instance and take a while to create, shared instance plans re-used the same RDS database instance and created new databases within the same instance, which completed almost instantly.
+
+This article will cover how to replicate the behavior of the deprecated shared instance plans using an 
+open-source service broker known as [`postgres-tinsmith`](https://github.com/markdboyd/cf-postgres-tinsmith).
+
+Please note that while this approach has been tested manually, the `postgres-tinsmith` repo is not
```

Comment:
This paragraph should move to the end. I'd also emphasize that this is a development aid, rather than a recommendation that all DBs used by production apps reside in the same RDS instance.

Reply:
I'm happy to do that, but are we worried that if move this content to the end that some people might miss it? Generally I would think we want to put important caveats/disclaimers up front in case people don't read all the way to the end